changes/test_tasks.py

```diff
@@ -311,6 +312,30 @@ def test_no_message_found(self, mock_create_outbound):
         )
         self.assertFalse(mock_create_outbound.called)
 
+    @mock.patch("changes.tasks.utils.ms_client.create_outbound")
+    @responses.activate
+    def test_timeout_message_sent_delivered(self, mock_create_outbound):
+        """
+        The task should send the correct outbound based on the delivered event.
+        """
+        self.create_outbound_lookup()
+        self.create_identity_lookup()
+
+        process_whatsapp_timeout_system_event("messageid", 1542844800, "undelivered")
```

Comment:
The issue with hard-coding the timestamp here is that, although this might work now, `datetime.now()` is always going to increase, so as time passes you'll get a bigger value for `datetime.now()`, but this value will stay the same, so eventually this test will fail, even though no code has changed.

This should rather be a value that is relative to `datetime.now()`

Reply:
thanks for pointing this out